packages/nc-gui/components/general/JoinCloud.vue

```diff
@@ -1,4 +1,37 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+const isDropdownOpen = ref(false)
+
+const descriptions: Array<{
+  title: string
+  highlight?: boolean
+  comingSoon?: boolean
+}> = [
+  {
+    title: 'SAML based Single Sign-On',
+    highlight: true,
+  },
+  {
+    title: 'Form view branding',
+  },
+  {
+    title: 'Personal views',
+  },
+
+  {
+    title: 'Extensions',
+    comingSoon: true,
+  },
+  {
+    title: 'Scripts',
+    comingSoon: true,
+  },
+  {
+    title: 'AI Integrations',
+    comingSoon: true,
+    highlight: true,
+  },
+]
+</script>
```

Comment:
Instead of hardcoding, we could fetch this data from the server, so we can update it in old instances in the future as well